src/controllers/music/musicrecommended.js

```diff
@@ -362,7 +362,7 @@ import Dashboard from '../../utils/dashboard';
             const containers = tabContent.querySelectorAll('.itemsContainer');
 
             for (let i = 0, length = containers.length; i < length; i++) {
-                setScrollClasses(containers[i], enableScrollX());
+                setScrollClasses(containers[i], enableScrollXmobile());
```

Comment:
We could probably just remove this function and use the `browser.mobile` value directly.